Jobs failing for python 3.5 & 3.6 on travis-ci builds

Hi All,
I am trying to build django-bitfield on travis-ci but jobs on python 3.5 & 3.6 are failing. Error encountered during setup. Below are the logs

python 3.5
------------
 travis_setup_postgresql
Starting PostgreSQL v9.6
sudo systemctl start postgresql@9.6-main
git.checkout
0.65s$ git clone --depth=50 --branch=master https://github.com/kishorkunal-raj/django-bitfield.git kishorkunal-raj/django-bitfield
0.01s0.00s$ source ~/virtualenv/python3.5/bin/activate
$ python --version
Python 3.5.6
$ pip --version
pip 18.1 from /home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/pip (python 3.5)
install
1.90s$ pip install flake8 tox-travis
before_script
0.22s$ psql -c 'create database bitfield;' -U postgres;
0.58s$ make lint
--> Linting Python files
PYFLAKES_NODOCTEST=1 flake8 bitfield
The command "make lint" exited with 0.
91.55s$ make test
--> Running Python tests
tox
GLOB sdist-make: /home/travis/build/kishorkunal-raj/django-bitfield/setup.py
py35-django110-sqlite create: /home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-sqlite
py35-django110-sqlite installdeps: pytest, psycopg2>=2.3, Django>=1.10,<1.11, pytest-django>=3.1
py35-django110-sqlite inst: /home/travis/build/kishorkunal-raj/django-bitfield/.tox/.tmp/package/1/django-bitfield-2.0.1.zip
py35-django110-sqlite installed: attrs==20.3.0,Django==1.10.8,django-bitfield==2.0.1,importlib-metadata==2.1.1,iniconfig==1.1.1,packaging==20.7,pathlib2==2.3.5,pluggy==0.13.1,psycopg2==2.8.6,py==1.9.0,pyparsing==2.4.7,pytest==6.1.2,pytest-django==4.1.0,six==1.15.0,toml==0.10.2,zipp==1.2.0
py35-django110-sqlite run-test-pre: PYTHONHASHSEED='1251796090'
py35-django110-sqlite run-test: commands[0] | py.test
============================= test session starts ==============================
platform linux -- Python 3.5.6, pytest-6.1.2, py-1.9.0, pluggy-0.13.1
cachedir: .tox/py35-django110-sqlite/.pytest_cache
rootdir: /home/travis/build/kishorkunal-raj/django-bitfield, configfile: setup.cfg
plugins: django-4.1.0
collected 29 items                                                             
bitfield/tests/tests.py EEEEEEEEEEEEEEEEEEEEEEEEEEEEE                    [100%]
==================================== ERRORS ====================================
_______________ ERROR at setup of BitHandlerTest.test_comparison _______________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-sqlite/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
________________ ERROR at setup of BitHandlerTest.test_defaults ________________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-sqlite/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
________________ ERROR at setup of BitHandlerTest.test_mutation ________________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-sqlite/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
____________ ERROR at setup of BitHandlerTest.test_nonzero_default _____________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-sqlite/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
______________________ ERROR at setup of BitTest.test_and ______________________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-sqlite/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
__________________ ERROR at setup of BitTest.test_comparison ___________________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-sqlite/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
______________________ ERROR at setup of BitTest.test_int ______________________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-sqlite/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
______________________ ERROR at setup of BitTest.test_or _______________________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-sqlite/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
______________________ ERROR at setup of BitTest.test_xor ______________________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-sqlite/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
__________________ ERROR at setup of BitFieldTest.test_basic ___________________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-sqlite/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
_____________ ERROR at setup of BitFieldTest.test_binary_capacity ______________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-sqlite/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
______________ ERROR at setup of BitFieldTest.test_default_value _______________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-sqlite/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
__________ ERROR at setup of BitFieldTest.test_defaults_as_key_names ___________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-sqlite/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
_____________ ERROR at setup of BitFieldTest.test_dictionary_init ______________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-sqlite/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
__________________ ERROR at setup of BitFieldTest.test_negate __________________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-sqlite/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
_____________ ERROR at setup of BitFieldTest.test_regression_1425 ______________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-sqlite/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
__________________ ERROR at setup of BitFieldTest.test_select __________________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-sqlite/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
________ ERROR at setup of BitFieldTest.test_select_complex_expression _________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-sqlite/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
__________________ ERROR at setup of BitFieldTest.test_update __________________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-sqlite/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
___________ ERROR at setup of BitFieldTest.test_update_with_handler ____________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-sqlite/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
_________ ERROR at setup of BitFieldSerializationTest.test_added_field _________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-sqlite/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
_ ERROR at setup of BitFieldSerializationTest.test_can_unserialize_bithandler __
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-sqlite/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
_____ ERROR at setup of BitFieldSerializationTest.test_pickle_integration ______
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-sqlite/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
____________ ERROR at setup of CompositeBitFieldTest.test_get_flag _____________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-sqlite/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
_____________ ERROR at setup of CompositeBitFieldTest.test_hasattr _____________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-sqlite/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
____________ ERROR at setup of CompositeBitFieldTest.test_set_flag _____________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-sqlite/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
_______________ ERROR at setup of BitFormFieldTest.test_form_new _______________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-sqlite/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
___________ ERROR at setup of BitFormFieldTest.test_form_new_invalid ___________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-sqlite/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
_____________ ERROR at setup of BitFormFieldTest.test_form_update ______________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-sqlite/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
=========================== short test summary info ============================
ERROR bitfield/tests/tests.py::BitHandlerTest::test_comparison - TypeError: s...
ERROR bitfield/tests/tests.py::BitHandlerTest::test_defaults - TypeError: set...
ERROR bitfield/tests/tests.py::BitHandlerTest::test_mutation - TypeError: set...
ERROR bitfield/tests/tests.py::BitHandlerTest::test_nonzero_default - TypeErr...
ERROR bitfield/tests/tests.py::BitTest::test_and - TypeError: setup_test_envi...
ERROR bitfield/tests/tests.py::BitTest::test_comparison - TypeError: setup_te...
ERROR bitfield/tests/tests.py::BitTest::test_int - TypeError: setup_test_envi...
ERROR bitfield/tests/tests.py::BitTest::test_or - TypeError: setup_test_envir...
ERROR bitfield/tests/tests.py::BitTest::test_xor - TypeError: setup_test_envi...
ERROR bitfield/tests/tests.py::BitFieldTest::test_basic - TypeError: setup_te...
ERROR bitfield/tests/tests.py::BitFieldTest::test_binary_capacity - TypeError...
ERROR bitfield/tests/tests.py::BitFieldTest::test_default_value - TypeError: ...
ERROR bitfield/tests/tests.py::BitFieldTest::test_defaults_as_key_names - Typ...
ERROR bitfield/tests/tests.py::BitFieldTest::test_dictionary_init - TypeError...
ERROR bitfield/tests/tests.py::BitFieldTest::test_negate - TypeError: setup_t...
ERROR bitfield/tests/tests.py::BitFieldTest::test_regression_1425 - TypeError...
ERROR bitfield/tests/tests.py::BitFieldTest::test_select - TypeError: setup_t...
ERROR bitfield/tests/tests.py::BitFieldTest::test_select_complex_expression
ERROR bitfield/tests/tests.py::BitFieldTest::test_update - TypeError: setup_t...
ERROR bitfield/tests/tests.py::BitFieldTest::test_update_with_handler - TypeE...
ERROR bitfield/tests/tests.py::BitFieldSerializationTest::test_added_field - ...
ERROR bitfield/tests/tests.py::BitFieldSerializationTest::test_can_unserialize_bithandler
ERROR bitfield/tests/tests.py::BitFieldSerializationTest::test_pickle_integration
ERROR bitfield/tests/tests.py::CompositeBitFieldTest::test_get_flag - TypeErr...
ERROR bitfield/tests/tests.py::CompositeBitFieldTest::test_hasattr - TypeErro...
ERROR bitfield/tests/tests.py::CompositeBitFieldTest::test_set_flag - TypeErr...
ERROR bitfield/tests/tests.py::BitFormFieldTest::test_form_new - TypeError: s...
ERROR bitfield/tests/tests.py::BitFormFieldTest::test_form_new_invalid - Type...
ERROR bitfield/tests/tests.py::BitFormFieldTest::test_form_update - TypeError...
============================== 29 errors in 0.18s ==============================
ERROR: InvocationError for command /home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-sqlite/bin/py.test (exited with code 1)
py35-django110-postgres create: /home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-postgres
py35-django110-postgres installdeps: pytest, psycopg2>=2.3, Django>=1.10,<1.11, pytest-django>=3.1
py35-django110-postgres inst: /home/travis/build/kishorkunal-raj/django-bitfield/.tox/.tmp/package/1/django-bitfield-2.0.1.zip
py35-django110-postgres installed: attrs==20.3.0,Django==1.10.8,django-bitfield==2.0.1,importlib-metadata==2.1.1,iniconfig==1.1.1,packaging==20.7,pathlib2==2.3.5,pluggy==0.13.1,psycopg2==2.8.6,py==1.9.0,pyparsing==2.4.7,pytest==6.1.2,pytest-django==4.1.0,six==1.15.0,toml==0.10.2,zipp==1.2.0
py35-django110-postgres run-test-pre: PYTHONHASHSEED='1251796090'
py35-django110-postgres run-test: commands[0] | py.test
============================= test session starts ==============================
platform linux -- Python 3.5.6, pytest-6.1.2, py-1.9.0, pluggy-0.13.1
cachedir: .tox/py35-django110-postgres/.pytest_cache
rootdir: /home/travis/build/kishorkunal-raj/django-bitfield, configfile: setup.cfg
plugins: django-4.1.0
collected 29 items                                                             
bitfield/tests/tests.py EEEEEEEEEEEEEEEEEEEEEEEEEEEEE                    [100%]
==================================== ERRORS ====================================
_______________ ERROR at setup of BitHandlerTest.test_comparison _______________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-postgres/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
________________ ERROR at setup of BitHandlerTest.test_defaults ________________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-postgres/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
________________ ERROR at setup of BitHandlerTest.test_mutation ________________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-postgres/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
____________ ERROR at setup of BitHandlerTest.test_nonzero_default _____________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-postgres/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
______________________ ERROR at setup of BitTest.test_and ______________________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-postgres/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
__________________ ERROR at setup of BitTest.test_comparison ___________________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-postgres/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
______________________ ERROR at setup of BitTest.test_int ______________________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-postgres/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
______________________ ERROR at setup of BitTest.test_or _______________________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-postgres/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
______________________ ERROR at setup of BitTest.test_xor ______________________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-postgres/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
__________________ ERROR at setup of BitFieldTest.test_basic ___________________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-postgres/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
_____________ ERROR at setup of BitFieldTest.test_binary_capacity ______________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-postgres/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
______________ ERROR at setup of BitFieldTest.test_default_value _______________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-postgres/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
__________ ERROR at setup of BitFieldTest.test_defaults_as_key_names ___________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-postgres/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
_____________ ERROR at setup of BitFieldTest.test_dictionary_init ______________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-postgres/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
__________________ ERROR at setup of BitFieldTest.test_negate __________________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-postgres/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
_____________ ERROR at setup of BitFieldTest.test_regression_1425 ______________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-postgres/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
__________________ ERROR at setup of BitFieldTest.test_select __________________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-postgres/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
________ ERROR at setup of BitFieldTest.test_select_complex_expression _________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-postgres/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
__________________ ERROR at setup of BitFieldTest.test_update __________________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-postgres/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
___________ ERROR at setup of BitFieldTest.test_update_with_handler ____________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-postgres/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
_________ ERROR at setup of BitFieldSerializationTest.test_added_field _________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-postgres/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
_ ERROR at setup of BitFieldSerializationTest.test_can_unserialize_bithandler __
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-postgres/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
_____ ERROR at setup of BitFieldSerializationTest.test_pickle_integration ______
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-postgres/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
____________ ERROR at setup of CompositeBitFieldTest.test_get_flag _____________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-postgres/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
_____________ ERROR at setup of CompositeBitFieldTest.test_hasattr _____________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-postgres/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
____________ ERROR at setup of CompositeBitFieldTest.test_set_flag _____________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-postgres/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
_______________ ERROR at setup of BitFormFieldTest.test_form_new _______________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-postgres/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
___________ ERROR at setup of BitFormFieldTest.test_form_new_invalid ___________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-postgres/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
_____________ ERROR at setup of BitFormFieldTest.test_form_update ______________
Traceback (most recent call last):
  File "/home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-postgres/lib/python3.5/site-packages/pytest_django/plugin.py", line 397, in django_test_environment
    setup_test_environment(debug=debug)
TypeError: setup_test_environment() got an unexpected keyword argument 'debug'
=========================== short test summary info ============================
ERROR bitfield/tests/tests.py::BitHandlerTest::test_comparison - TypeError: s...
ERROR bitfield/tests/tests.py::BitHandlerTest::test_defaults - TypeError: set...
ERROR bitfield/tests/tests.py::BitHandlerTest::test_mutation - TypeError: set...
ERROR bitfield/tests/tests.py::BitHandlerTest::test_nonzero_default - TypeErr...
ERROR bitfield/tests/tests.py::BitTest::test_and - TypeError: setup_test_envi...
ERROR bitfield/tests/tests.py::BitTest::test_comparison - TypeError: setup_te...
ERROR bitfield/tests/tests.py::BitTest::test_int - TypeError: setup_test_envi...
ERROR bitfield/tests/tests.py::BitTest::test_or - TypeError: setup_test_envir...
ERROR bitfield/tests/tests.py::BitTest::test_xor - TypeError: setup_test_envi...
ERROR bitfield/tests/tests.py::BitFieldTest::test_basic - TypeError: setup_te...
ERROR bitfield/tests/tests.py::BitFieldTest::test_binary_capacity - TypeError...
ERROR bitfield/tests/tests.py::BitFieldTest::test_default_value - TypeError: ...
ERROR bitfield/tests/tests.py::BitFieldTest::test_defaults_as_key_names - Typ...
ERROR bitfield/tests/tests.py::BitFieldTest::test_dictionary_init - TypeError...
ERROR bitfield/tests/tests.py::BitFieldTest::test_negate - TypeError: setup_t...
ERROR bitfield/tests/tests.py::BitFieldTest::test_regression_1425 - TypeError...
ERROR bitfield/tests/tests.py::BitFieldTest::test_select - TypeError: setup_t...
ERROR bitfield/tests/tests.py::BitFieldTest::test_select_complex_expression
ERROR bitfield/tests/tests.py::BitFieldTest::test_update - TypeError: setup_t...
ERROR bitfield/tests/tests.py::BitFieldTest::test_update_with_handler - TypeE...
ERROR bitfield/tests/tests.py::BitFieldSerializationTest::test_added_field - ...
ERROR bitfield/tests/tests.py::BitFieldSerializationTest::test_can_unserialize_bithandler
ERROR bitfield/tests/tests.py::BitFieldSerializationTest::test_pickle_integration
ERROR bitfield/tests/tests.py::CompositeBitFieldTest::test_get_flag - TypeErr...
ERROR bitfield/tests/tests.py::CompositeBitFieldTest::test_hasattr - TypeErro...
ERROR bitfield/tests/tests.py::CompositeBitFieldTest::test_set_flag - TypeErr...
ERROR bitfield/tests/tests.py::BitFormFieldTest::test_form_new - TypeError: s...
ERROR bitfield/tests/tests.py::BitFormFieldTest::test_form_new_invalid - Type...
ERROR bitfield/tests/tests.py::BitFormFieldTest::test_form_update - TypeError...
============================== 29 errors in 0.17s ==============================
ERROR: InvocationError for command /home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django110-postgres/bin/py.test (exited with code 1)
py35-django111-sqlite create: /home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django111-sqlite
py35-django111-sqlite installdeps: pytest, psycopg2>=2.3, Django>=1.11,<1.12, pytest-django>=3.1
py35-django111-sqlite inst: /home/travis/build/kishorkunal-raj/django-bitfield/.tox/.tmp/package/1/django-bitfield-2.0.1.zip
py35-django111-sqlite installed: attrs==20.3.0,Django==1.11.29,django-bitfield==2.0.1,importlib-metadata==2.1.1,iniconfig==1.1.1,packaging==20.7,pathlib2==2.3.5,pluggy==0.13.1,psycopg2==2.8.6,py==1.9.0,pyparsing==2.4.7,pytest==6.1.2,pytest-django==4.1.0,pytz==2020.4,six==1.15.0,toml==0.10.2,zipp==1.2.0
py35-django111-sqlite run-test-pre: PYTHONHASHSEED='1251796090'
py35-django111-sqlite run-test: commands[0] | py.test
============================= test session starts ==============================
platform linux -- Python 3.5.6, pytest-6.1.2, py-1.9.0, pluggy-0.13.1
cachedir: .tox/py35-django111-sqlite/.pytest_cache
rootdir: /home/travis/build/kishorkunal-raj/django-bitfield, configfile: setup.cfg
plugins: django-4.1.0
collected 29 items                                                             
bitfield/tests/tests.py .............................                    [100%]
============================== 29 passed in 0.21s ==============================
py35-django111-postgres create: /home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django111-postgres
py35-django111-postgres installdeps: pytest, psycopg2>=2.3, Django>=1.11,<1.12, pytest-django>=3.1
py35-django111-postgres inst: /home/travis/build/kishorkunal-raj/django-bitfield/.tox/.tmp/package/1/django-bitfield-2.0.1.zip
py35-django111-postgres installed: attrs==20.3.0,Django==1.11.29,django-bitfield==2.0.1,importlib-metadata==2.1.1,iniconfig==1.1.1,packaging==20.7,pathlib2==2.3.5,pluggy==0.13.1,psycopg2==2.8.6,py==1.9.0,pyparsing==2.4.7,pytest==6.1.2,pytest-django==4.1.0,pytz==2020.4,six==1.15.0,toml==0.10.2,zipp==1.2.0
py35-django111-postgres run-test-pre: PYTHONHASHSEED='1251796090'
py35-django111-postgres run-test: commands[0] | py.test
============================= test session starts ==============================
platform linux -- Python 3.5.6, pytest-6.1.2, py-1.9.0, pluggy-0.13.1
cachedir: .tox/py35-django111-postgres/.pytest_cache
rootdir: /home/travis/build/kishorkunal-raj/django-bitfield, configfile: setup.cfg
plugins: django-4.1.0
collected 29 items                                                             
bitfield/tests/tests.py .............................                    [100%]
============================== 29 passed in 0.48s ==============================
py35-django20-sqlite create: /home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django20-sqlite
py35-django20-sqlite installdeps: pytest, psycopg2>=2.3, Django>=2.0,<2.1, pytest-django>=3.1
py35-django20-sqlite inst: /home/travis/build/kishorkunal-raj/django-bitfield/.tox/.tmp/package/1/django-bitfield-2.0.1.zip
py35-django20-sqlite installed: attrs==20.3.0,Django==2.0.13,django-bitfield==2.0.1,importlib-metadata==2.1.1,iniconfig==1.1.1,packaging==20.7,pathlib2==2.3.5,pluggy==0.13.1,psycopg2==2.8.6,py==1.9.0,pyparsing==2.4.7,pytest==6.1.2,pytest-django==4.1.0,pytz==2020.4,six==1.15.0,toml==0.10.2,zipp==1.2.0
py35-django20-sqlite run-test-pre: PYTHONHASHSEED='1251796090'
py35-django20-sqlite run-test: commands[0] | py.test
============================= test session starts ==============================
platform linux -- Python 3.5.6, pytest-6.1.2, py-1.9.0, pluggy-0.13.1
cachedir: .tox/py35-django20-sqlite/.pytest_cache
rootdir: /home/travis/build/kishorkunal-raj/django-bitfield, configfile: setup.cfg
plugins: django-4.1.0
collected 29 items                                                             
bitfield/tests/tests.py .............................                    [100%]
=============================== warnings summary ===============================
.tox/py35-django20-sqlite/lib/python3.5/distutils/__init__.py:1
  /home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django20-sqlite/lib/python3.5/distutils/__init__.py:1: PendingDeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
    import imp
-- Docs: https://docs.pytest.org/en/stable/warnings.html
======================== 29 passed, 1 warning in 0.20s =========================
py35-django20-postgres create: /home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django20-postgres
py35-django20-postgres installdeps: pytest, psycopg2>=2.3, Django>=2.0,<2.1, pytest-django>=3.1
py35-django20-postgres inst: /home/travis/build/kishorkunal-raj/django-bitfield/.tox/.tmp/package/1/django-bitfield-2.0.1.zip
py35-django20-postgres installed: attrs==20.3.0,Django==2.0.13,django-bitfield==2.0.1,importlib-metadata==2.1.1,iniconfig==1.1.1,packaging==20.7,pathlib2==2.3.5,pluggy==0.13.1,psycopg2==2.8.6,py==1.9.0,pyparsing==2.4.7,pytest==6.1.2,pytest-django==4.1.0,pytz==2020.4,six==1.15.0,toml==0.10.2,zipp==1.2.0
py35-django20-postgres run-test-pre: PYTHONHASHSEED='1251796090'
py35-django20-postgres run-test: commands[0] | py.test
============================= test session starts ==============================
platform linux -- Python 3.5.6, pytest-6.1.2, py-1.9.0, pluggy-0.13.1
cachedir: .tox/py35-django20-postgres/.pytest_cache
rootdir: /home/travis/build/kishorkunal-raj/django-bitfield, configfile: setup.cfg
plugins: django-4.1.0
collected 29 items                                                             
bitfield/tests/tests.py .............................                    [100%]
=============================== warnings summary ===============================
.tox/py35-django20-postgres/lib/python3.5/distutils/__init__.py:1
  /home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django20-postgres/lib/python3.5/distutils/__init__.py:1: PendingDeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
    import imp
-- Docs: https://docs.pytest.org/en/stable/warnings.html
======================== 29 passed, 1 warning in 0.47s =========================
py35-django21-sqlite create: /home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django21-sqlite
py35-django21-sqlite installdeps: pytest, psycopg2>=2.3, Django>=2.1,<2.2, pytest-django>=3.1
py35-django21-sqlite inst: /home/travis/build/kishorkunal-raj/django-bitfield/.tox/.tmp/package/1/django-bitfield-2.0.1.zip
py35-django21-sqlite installed: attrs==20.3.0,Django==2.1.15,django-bitfield==2.0.1,importlib-metadata==2.1.1,iniconfig==1.1.1,packaging==20.7,pathlib2==2.3.5,pluggy==0.13.1,psycopg2==2.8.6,py==1.9.0,pyparsing==2.4.7,pytest==6.1.2,pytest-django==4.1.0,pytz==2020.4,six==1.15.0,toml==0.10.2,zipp==1.2.0
py35-django21-sqlite run-test-pre: PYTHONHASHSEED='1251796090'
py35-django21-sqlite run-test: commands[0] | py.test
============================= test session starts ==============================
platform linux -- Python 3.5.6, pytest-6.1.2, py-1.9.0, pluggy-0.13.1
cachedir: .tox/py35-django21-sqlite/.pytest_cache
rootdir: /home/travis/build/kishorkunal-raj/django-bitfield, configfile: setup.cfg
plugins: django-4.1.0
collected 29 items                                                             
bitfield/tests/tests.py .............................                    [100%]
=============================== warnings summary ===============================
.tox/py35-django21-sqlite/lib/python3.5/distutils/__init__.py:1
  /home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django21-sqlite/lib/python3.5/distutils/__init__.py:1: PendingDeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
    import imp
-- Docs: https://docs.pytest.org/en/stable/warnings.html
======================== 29 passed, 1 warning in 0.20s =========================
py35-django21-postgres create: /home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django21-postgres
py35-django21-postgres installdeps: pytest, psycopg2>=2.3, Django>=2.1,<2.2, pytest-django>=3.1
py35-django21-postgres inst: /home/travis/build/kishorkunal-raj/django-bitfield/.tox/.tmp/package/1/django-bitfield-2.0.1.zip
py35-django21-postgres installed: attrs==20.3.0,Django==2.1.15,django-bitfield==2.0.1,importlib-metadata==2.1.1,iniconfig==1.1.1,packaging==20.7,pathlib2==2.3.5,pluggy==0.13.1,psycopg2==2.8.6,py==1.9.0,pyparsing==2.4.7,pytest==6.1.2,pytest-django==4.1.0,pytz==2020.4,six==1.15.0,toml==0.10.2,zipp==1.2.0
py35-django21-postgres run-test-pre: PYTHONHASHSEED='1251796090'
py35-django21-postgres run-test: commands[0] | py.test
============================= test session starts ==============================
platform linux -- Python 3.5.6, pytest-6.1.2, py-1.9.0, pluggy-0.13.1
cachedir: .tox/py35-django21-postgres/.pytest_cache
rootdir: /home/travis/build/kishorkunal-raj/django-bitfield, configfile: setup.cfg
plugins: django-4.1.0
collected 29 items                                                             
bitfield/tests/tests.py .............................                    [100%]
=============================== warnings summary ===============================
.tox/py35-django21-postgres/lib/python3.5/distutils/__init__.py:1
  /home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django21-postgres/lib/python3.5/distutils/__init__.py:1: PendingDeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
    import imp
-- Docs: https://docs.pytest.org/en/stable/warnings.html
======================== 29 passed, 1 warning in 0.47s =========================
py35-django22-sqlite create: /home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django22-sqlite
py35-django22-sqlite installdeps: pytest, psycopg2>=2.3, Django>=2.2,<2.3, pytest-django>=3.1
py35-django22-sqlite inst: /home/travis/build/kishorkunal-raj/django-bitfield/.tox/.tmp/package/1/django-bitfield-2.0.1.zip
py35-django22-sqlite installed: attrs==20.3.0,Django==2.2.17,django-bitfield==2.0.1,importlib-metadata==2.1.1,iniconfig==1.1.1,packaging==20.7,pathlib2==2.3.5,pluggy==0.13.1,psycopg2==2.8.6,py==1.9.0,pyparsing==2.4.7,pytest==6.1.2,pytest-django==4.1.0,pytz==2020.4,six==1.15.0,sqlparse==0.4.1,toml==0.10.2,zipp==1.2.0
py35-django22-sqlite run-test-pre: PYTHONHASHSEED='1251796090'
py35-django22-sqlite run-test: commands[0] | py.test
============================= test session starts ==============================
platform linux -- Python 3.5.6, pytest-6.1.2, py-1.9.0, pluggy-0.13.1
cachedir: .tox/py35-django22-sqlite/.pytest_cache
rootdir: /home/travis/build/kishorkunal-raj/django-bitfield, configfile: setup.cfg
plugins: django-4.1.0
collected 29 items                                                             
bitfield/tests/tests.py .............................                    [100%]
=============================== warnings summary ===============================
.tox/py35-django22-sqlite/lib/python3.5/distutils/__init__.py:1
  /home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django22-sqlite/lib/python3.5/distutils/__init__.py:1: PendingDeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
    import imp
-- Docs: https://docs.pytest.org/en/stable/warnings.html
======================== 29 passed, 1 warning in 0.20s =========================
py35-django22-postgres create: /home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django22-postgres
py35-django22-postgres installdeps: pytest, psycopg2>=2.3, Django>=2.2,<2.3, pytest-django>=3.1
py35-django22-postgres inst: /home/travis/build/kishorkunal-raj/django-bitfield/.tox/.tmp/package/1/django-bitfield-2.0.1.zip
py35-django22-postgres installed: attrs==20.3.0,Django==2.2.17,django-bitfield==2.0.1,importlib-metadata==2.1.1,iniconfig==1.1.1,packaging==20.7,pathlib2==2.3.5,pluggy==0.13.1,psycopg2==2.8.6,py==1.9.0,pyparsing==2.4.7,pytest==6.1.2,pytest-django==4.1.0,pytz==2020.4,six==1.15.0,sqlparse==0.4.1,toml==0.10.2,zipp==1.2.0
py35-django22-postgres run-test-pre: PYTHONHASHSEED='1251796090'
py35-django22-postgres run-test: commands[0] | py.test
============================= test session starts ==============================
platform linux -- Python 3.5.6, pytest-6.1.2, py-1.9.0, pluggy-0.13.1
cachedir: .tox/py35-django22-postgres/.pytest_cache
rootdir: /home/travis/build/kishorkunal-raj/django-bitfield, configfile: setup.cfg
plugins: django-4.1.0
collected 29 items                                                             
bitfield/tests/tests.py .............................                    [100%]
=============================== warnings summary ===============================
.tox/py35-django22-postgres/lib/python3.5/distutils/__init__.py:1
  /home/travis/build/kishorkunal-raj/django-bitfield/.tox/py35-django22-postgres/lib/python3.5/distutils/__init__.py:1: PendingDeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
    import imp
-- Docs: https://docs.pytest.org/en/stable/warnings.html
======================== 29 passed, 1 warning in 0.46s =========================
___________________________________ summary ____________________________________
ERROR:   py35-django110-sqlite: commands failed
ERROR:   py35-django110-postgres: commands failed
  py35-django111-sqlite: commands succeeded
  py35-django111-postgres: commands succeeded
  py35-django20-sqlite: commands succeeded
  py35-django20-postgres: commands succeeded
  py35-django21-sqlite: commands succeeded
  py35-django21-postgres: commands succeeded
  py35-django22-sqlite: commands succeeded
  py35-django22-postgres: commands succeeded
Makefile:8: recipe for target 'test' failed
make: *** [test] Error 1
traviThe command "make test" exited with 2.
Done. Your build exited with 1.


Please help me to fix the issue

--- a/.travis.yml
+++ b/.travis.yml
@@ -5,6 +5,8 @@ python:
 - '3.5'
 - '3.6'
 - '3.7'
+- '3.8'
+- '3.9'
 install: pip install flake8 tox-travis
 
 services:

--- a/tox.ini
+++ b/tox.ini
@@ -4,7 +4,7 @@
 # https://docs.djangoproject.com/en/dev/faq/install/#what-python-version-can-i-use-with-django
 envlist =
     py{27,34,35,36}-django{110,111}-{sqlite,postgres}, py{34,35,36,37}-django{20}-{sqlite,postgres},
-    py{35,36,37}-django{21,22}-{sqlite,postgres}, py{36,37}-django{30}-{sqlite,postgres}
+    py{35,36,37}-django{21,22}-{sqlite,postgres}, py{36,37}-django{30,31}-{sqlite,postgres}
 
 [testenv]
 commands =
@@ -25,6 +25,8 @@ deps =
   django22: pytest-django>=3.1
   django30: Django>=3.0,<3.1
   django30: pytest-django>=3.1
+  django31: Django>=3.1,<3.2
+  django31: pytest-django>=3.1
 setenv =
   sqlite: DB=sqlite
   postgres: DB=postgres

--- a/tox.ini
+++ b/tox.ini
@@ -3,8 +3,8 @@
 # Taken from:
 # https://docs.djangoproject.com/en/dev/faq/install/#what-python-version-can-i-use-with-django
 envlist =
-    py{27,34,35,36}-django{110,111}-{sqlite,postgres}, py{34,35,36,37}-django{20}-{sqlite,postgres},
-    py{35,36,37}-django{21,22}-{sqlite,postgres}, py{36,37}-django{30,31}-{sqlite,postgres}
+    py{27,34,35,36}-django{110,111}-{sqlite,postgres}, py{34,35,36,37,38,39}-django{20}-{sqlite,postgres},
+    py{35,36,37,38,39}-django{21,22}-{sqlite,postgres}, py{36,37,38,39}-django{30,31}-{sqlite,postgres}
 
 [testenv]
 commands =

--- a/tox.ini
+++ b/tox.ini
@@ -14,9 +14,9 @@ deps =
   pytest
   psycopg2>=2.3
   django110: Django>=1.10,<1.11
-  django110: pytest-django>=3.1
+  django110: pytest-django>=3.1,<4.0
   django111: Django>=1.11,<1.12
-  django111: pytest-django>=3.1
+  django111: pytest-django>=3.1,<4.0
   django20: Django>=2.0,<2.1
   django20: pytest-django>=3.1
   django21: Django>=2.1,<2.2

